integrationTests/chainSimulator/bridge/esdtSafe_test.go

```diff
@@ -515,6 +515,81 @@ func TestChainSimulator_ExecuteWithTransferDataFails(t *testing.T) {
 	}
 }
 
+// main chain deposit and execute with no payment, only transfer data
+// 1. deposit no payment, no transfer data - should fail
```

Comment:
Here and below everywhere in the comments, pls rephrase to smth like this:
```suggestion
// 1. call deposit endpoint with no payment and no transfer data - should fail
```

Reply:
You missed the comments below (like L573, 577, etc...)